src/sql/parts/profiler/editor/profilerInput.ts

```diff
@@ -268,4 +252,31 @@ export class ProfilerInput extends EditorInput implements IProfilerSession {
 		}
 
 	}
+
+	confirmSave(): TPromise<ConfirmResult> {
+		if (this._state.isRunning || this.state.isPaused) {
```

Comment:
why does the first use `_state` but the second uses `state`? is there a difference?

Reply:
💯  I didn't even notice that :) they are the same, state is the public property, a wrapper for _state. I will push another commit for this.